src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -19,328 +19,330 @@
-                  <span>{{ filedLabel('FILED AND PAID', filing) }}</span>
-                </div>
-
-                <!-- details (comments) button -->
-                <div v-if="filing.comments.length > 0" class="filing-subtitle">
-                  <v-btn
-                    class="comments-btn"
-                    outlined
-                    color="primary"
-                    :ripple=false
-                    @click.stop="togglePanel(index)"
-                  >
-                    <v-icon small left style="padding-top: 2px">mdi-message-reply</v-icon>
-                    Detail{{filing.comments.length > 1 ? "s" : ""}} ({{filing.comments.length}})
-                  </v-btn>
-                </div>
-              </div> <!-- end of subtitle -->
-            </div> <!-- end of filing-label -->
-
-            <!-- the action button/menu -->
-            <div class="filing-item__actions">
-              <v-btn
-                class="expand-btn"
-                outlined
-                :ripple=false
-                @click.stop="togglePanel(index)"
-              >
-                <span v-if="filing.isColinFiling || filing.isPaperFiling">
-                  {{ (panel === index) ? "Close" : "Request a Copy" }}</span>
-                <span v-else>{{ (panel === index) ? "Hide Documents" : "View Documents" }}</span>
-              </v-btn>
-
-              <!-- the drop-down menu -->
-              <v-menu offset-y left transition="slide-y-transition" v-if="isRoleStaff">
-                <template v-slot:activator="{ on }">
-                  <v-btn text v-on="on" class="menu-btn">
-                    <v-icon>mdi-menu-down</v-icon>
-                  </v-btn>
-                </template>
-                <v-list dense>
-                  <v-list-item-group color="primary">
-                    <v-list-item :disabled="disableCorrection(filing)">
-                      <v-list-item-icon>
-                        <v-icon>mdi-file-document-edit-outline</v-icon>
-                      </v-list-item-icon>
-                      <v-list-item-title
-                        class="file-correction-item"
-                        @click="correctThisFiling(filing)"
-                      >
-                        File a Correction
-                      </v-list-item-title>
-                    </v-list-item>
-
-                    <v-list-item>
-                      <v-list-item-icon>
-                        <v-icon>mdi-comment-plus</v-icon>
-                      </v-list-item-icon>
-                      <v-list-item-title
-                        class="add-detail-comment-item"
-                        @click="showCommentDialog(filing.filingId)"
-                      >
-                        Add Detail
-                      </v-list-item-title>
-                    </v-list-item>
-                  </v-list-item-group>
-                </v-list>
-              </v-menu>
+    <div class="scrollable-container" style="max-height: 1000px">
```

Comment:
Actually, I don't mind the style property here, since it's the whole point for this div.

Reply:
Thanks!